src/app/utils/query.ts

```diff
@@ -15,6 +15,23 @@ export function handleRetries(failureCount: number, error: unknown): boolean {
   return failureCount < 3;
 }
 
-export const queryClient = new QueryClient();
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      // time before garbage collection of query data
+      cacheTime: 5 * 60 * 1000, // 5 min
+
+      // increase this for specific queries to reduce refetches
+      staleTime: 0,
+
+      // at least one of these are required to refetch stale queries
+      // refetchInterval
+      // refetchIntervalInBackground
```

Comment:
the comment mentions that at least one of the refetchInterval or refetchIntervalInBackground are required. but then we don't use any. is that ok?

Reply:
Oh not clear, but it's meant to include the ones below which are not commented